README.md

```diff
@@ -138,66 +138,62 @@ paths:
             '*/*':
               schema:
                 type: string
+                example: hello world
 servers:
-  - url: '{schema}://{host}/{basePath}'
-    variables:
-      schema:
-        default: http
-        enum:
-          - http
-          - https
-      host:
-        default: stoplight.io
-        enum:
-          - stoplight.io
-          - dev.stoplight.io
-      basePath:
-        default: api
+  - url: https://stoplight.io/api
+    name: Production
+  - url: https://stag.stoplight.io/api
+    name: Staging
 ```
 
 You can make a request enforcing server validation by providing the `__server` query string parameter:
 
 ```bash
-curl -X GET localhost:4010/pet?__server=http://stoplight.io/api
-# This will return a 200 response
+curl -X GET localhost:4010/pet?__server=https://stoplight.io/api
```

Comment:
1. Shouldn't we put the protocol before the URL?
2. GET is implicitly set by curl, we can probably remove it.

```suggestion
curl http://localhost:4010/pet?__server=https://stoplight.io/api
```